### ISSUE TITLE
Record field containing "\"" causes parsing to fail

### ISSUE THREAD

**Issue by MartinSStewart** [layflags/elm-monitor#7]
If the model has a string field with the text `\"` in it then 
```
Expecting Keyword True at row 1, col 1
Expecting Keyword False at row 1, col 1
Expecting Symbol () at row 1, col 1
Expecting Symbol ' at row 1, col 1
Expecting Symbol " at row 1, col 1
Expecting Symbol < at row 1, col 1
Expecting { at row 1, col 1
Expecting [ at row 1, col 1
Expecting , at row 1, col 386
Expecting } at row 1, col 386
Expecting Variable at row 1, col 1
Expecting Symbol - at row 1, col 1
Expecting Number at row 1, col 1
```
gets printed to the console on every update.

Edit: So for example, `type alias Model = { someText : String}` and `initialModel = { someText = "\""}`

Edit2: This occurred in version 0.0.8

**Comment by ChristophP** [comment]
@MartinSStewart I tried a fix bit it was trickier than I thought, then I got confused a bit. :-) I will try to fix it in the next couple of days.

**Comment by layflags** [comment]
@MartinSStewart I just published v0.0.9 ... thx for reporting! Danke dir @ChristophP 👍
